### PR TITLE
do not walk &rtp if abs path to snips specified

### PR DESCRIPTION
**Pull request by seletskiy**

### CODE REVIEW

File: doc/UltiSnips.txt [hunk 1]
```diff
@@ -350,6 +357,12 @@ over the global variable.
 
 |UltiSnips-adding-snippets| explains which files are parsed for a given filetype.
 
+If only one directory is specified in this variable and this directory is
+specified by absolute path, UltiSnips will not look for snippets in
+&runtimepath, which can lead to significant speedup. So, the common case is:
```
Comment (SirVer) [inline doc/UltiSnips.txt]: mention that you will not see/use snippets shipped with other plugins then, only the ones in this directory.

**Comment by SirVer** [comment]
One more nit, otherwise lgtm.

**Comment by SirVer** [comment]
ping?

**Comment by seletskiy** [comment]
Oh, sorry. Will fix today.

**Comment by seletskiy** [comment]
@Sirver: rebased on latest master and added note you mentioned.

**Comment by SirVer** [comment]
And merged. Thanks for the contribution!

**Pull request merged**